libs/hooks/use-registration/src/lib/hooks-use-registration.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { SharedRegistration } from '@ppm/shared/registration';
+import {
+  useParams
+} from "react-router-dom";
+import { authorizationActions, authorizationSelectors } from '@ppm/data-access/authorization';
+import { useDispatch, useSelector } from 'react-redux';
+import { RegistrationInterface, PrivateRoutesPath } from '@ppm/common/main';
+import { SharedRegistrationVerification } from '@ppm/shared/registration-verification';
+import { FeaturesAuthorization } from '@ppm/features/authorization';
+import { createStructuredSelector } from 'reselect';
+import { clearRegistrationData } from '@ppm/data-access/local-storage';
+
+function getSteps() {
```

Comment:
Why function?! You could simple use `const steps = ['Registration', 'Verification', 'Login'];` Now you are calling a function which returns a static data.